### PR TITLE
python: python examples rewrite.

### PR DESCRIPTION
**Pull request by cristi-iacob**
Rewrote iio_readdev.py and iio_writedev.py in a more pythonic fashion.

Signed-off-by: Cristi Iacob <cristian.iacob@analog.com>

### CODE REVIEW

File: bindings/python/examples/iio_writedev.py [hunk 1]
```diff
@@ -16,208 +16,218 @@
...
-def main():
-    read_arguments()
+    def write(self):
+        """Push data into the buffer."""
+        app_running = True
+        num_samples = self.arguments.num_samples
 
-    ctx = create_context(scan_for_context, arg_uri, arg_ip)
+        while app_running:
+            bytes_to_read = self.buffer._length if num_samples == 0 \
```
Comment (tfcollins) [inline bindings/python/examples/iio_writedev.py:199]: use len(self.buffer) instead

File: bindings/python/examples/iio_writedev.py [hunk 2]
```diff
@@ -16,208 +16,218 @@
...
+                else min(self.buffer._length, num_samples * self.device.sample_size)
```
Comment (tfcollins) [inline bindings/python/examples/iio_writedev.py:200]: use len(self.buffer) instead

**Comment by rgetz** [comment]
#501 should fix some of the issues codacy issues. It just need to be approved and committed before this is pushed.

**Comment by rgetz** [comment]
This was done as part of #503

**Pull request closed without merging**